MAUI/Cartesian-Charts/neumorphicUI-column-chart.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: Neumorphic UI Column Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here about how to create the Neumorphic UI chart types using Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Neumorphic UI Chart using Syncfusion .NET MAUI Column Chart
+
+Learn to create visually stunning Neumorphic charts in using Syncfusion [.NET MAUI Column Chart](). Follow a step-by-step tutorial utilizing a custom class that inherits from `IDrawable` and `Graphics view` for a seamless blend of design and data representation in your application.
```

Comment:
@DhanarajRajendran 
Modify the first line as 
  Learn to create visually stunning Neumorphism UI charts using Syncfusion [.NET MAUI Column Chart]

Reply:
Addressed the comments.